execution/evm/vm.go

```diff
@@ -1128,3 +1124,28 @@ func transfer(from, to acm.MutableAccount, amount uint64) error {
 	}
 	return nil
 }
+
+// Dump the bytecode being sent to the EVM in the current working directory
+func dumpTokens(txHash []byte, caller, callee acm.Account, code []byte) {
```

Comment:
What's this useful for?

Reply:
Provides human-readable bytecode for each tx that is committed - can be convenient for comparing with other generated bytecode and can be clearer than the line-by-line opcode debugging - which also only prints the path that was executed rather than everything. That also doesn't have the tx and callee so obviously. Ultimately this could be combined/enhanced into a single debug, but I found both of these useful so rather than killing the debug I've made them available via options that seems like a fairly clean way of keeping them in the codebase.